subsys/bluetooth/controller/ll_sw/ull_llcp_hci.c

```diff
@@ -101,11 +101,11 @@ uint8_t ll_apto_get(uint16_t handle, uint16_t *apto)
 		return BT_HCI_ERR_UNKNOWN_CONN_ID;
 	}
 
-	return BT_HCI_ERR_CMD_DISALLOWED;
-	/* EGON: for future reference
-	 *
-	 *	*apto = conn->apto_reload * conn->lll.interval * 125U / 1000;
-	 */
+	// apto unit is 10ms
+	*apto = conn->apto_reload * conn->lll.interval * (CONN_INT_UNIT_US/10) / 1000;
+
+	return 0;
```

Comment:
This will fail test_hci_apto in tests/bluetooth/controller/ctrl_hci/bluetooth.controller.ctrl_hci.test

Reply:
fixed